cypress/integration/change_ecm.spec.js

```diff
@@ -0,0 +1,94 @@
+describe("Change ECM", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/supervision/deputies/public-authority/deputy/1/change-ecm");
+    });
+
+    it("has headers for different sections", () => {
+        cy.get("h1").should("contain", "Change Executive Case Manager");
+    })
+
+    it("leaves current ecm blank if none is set", () => {
+        cy.get(".govuk-body").should("contain", "Current ECM:");
+        cy.get(".govuk-label").should("contain", "Enter an Executive Case Manager name");
+    });
+
+    it("shows ecm if is set", () => {
+        cy.visit("/supervision/deputies/public-authority/deputy/2/change-ecm");
+        cy.get(".govuk-body").should("contain", "Current ECM:");
+        cy.get(".govuk-body").should("contain", "LayTeam1 User2");
+    });
+
+    it("has a drop down populated with members of the PA Deputy Team", () => {
+        cy.get("#select-ecm").type('S');
+        cy.get("#select-ecm__listbox").find('li').should('have.length', 3);
+        cy.get("#select-ecm").type('now');
+        cy.get("#select-ecm__listbox").find('li').should('have.length', 1);
+    })
+
+    it("directs me back to dashboard page if I press cancel", () => {
+        cy.get(".govuk-link").should("contain", "Cancel").click();
+        cy.url().should('not.include', '/change-ecm')
+        cy.get("h1").should("contain", "Dashboard");
+    })
+
+    it("allows me to fill in and submit the ecm form", () => {
+        cy.visit("/supervision/deputies/public-authority/deputy/1/change-ecm");
+        cy.setCookie("success-route", "ecm");
+        cy.get("#select-ecm").type('S');
+        cy.contains("#select-ecm__listbox", 'Jon Snow').click();
+        cy.get('form').submit();
+        cy.url().should("contain", "?success=ecm");
+        cy.get("h1").should("contain", "Dashboard");
+        cy.get(".moj-banner--success").should("contain", "Ecm changed to" );
+    })
+
+    it("has a timeline event for when an ecm is automatically allocated on deputy creation", () => {
```

Comment:
Put the timeline tests in a separate describe block. Otherwise you are navigating to the change ecm page and then navigating to the timeline